### PR TITLE
Pitchouu pitchouu

### PR DESCRIPTION
**Pull request by ppepos**
Add fireworks on successful submissions.

I put the firework code in the vendors directory although it's custom. It has pretty much nothing to do with the angular app and it will keep it somewhat simpler for human kind if it is kept out of the way. The call to the fireworks is in the submission module though.

I tweaked the color scheme to respect _MD_'s color palette and added a custom trigger function. Hope that adds some fun.

Depends on #143 
(First 2 commits come from there)

### CODE REVIEW

**Comment by privat** [comment]
nice! But we see nothing if the windows is scrolled down.

File: www/vendors/fireworks/fireworks.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    }).replay();
+    smallCircle.tune({
+        x: randX,
+        y: randY,
+				fill: randColor
+    }).replay();
+};
+
+var launchFireworks = function () {
+	var count = getRandIntRange(4, 10);
```
Comment (privat) [inline www/vendors/fireworks/fireworks.js]: The number of fires should be constant with only a slight variation. 8 to 10 for instance

**Review by privat — APPROVED** [review]

File: www/vendors/fireworks/fireworks.js (hunk 1 above)
Comment (ppepos) [reply www/vendors/fireworks/fireworks.js]: Sure... I don't mind =P

**Comment by ppepos** [comment]
I'll trigger a scrollTop on submission... I think that's the simplest/cleanest way of dealing with the unknown location of the viewport.

**Review by privat — APPROVED** [review]

**Pull request merged**